src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,49 @@
+.page {
+    margin: 0;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.calendar {
+    display: flex;
+    flex-wrap: wrap;
+    width: $calendar-width;
+    padding: 0;
+    margin: auto;
+    gap: $gap;
+
+    &__day {
+        box-sizing: border-box;
+        width: $day-size;
+        height: $day-size;
+        background-color: $day-color;
+        border: $day-border;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        font-family: Arial, sans-serif;
+        font-size: $font-size;
```

Comment:
Ensure that the variable `$font-size` is defined. If it's not defined in this file, it should be imported from another SCSS file.